.gitignore

```diff
@@ -4,6 +4,7 @@
 Toolchains/
 config/github.json
 *.xcodeproj
+.vscode/launch.json
```

Comment:
The `launch.json` was automatically generated by VSCode extension, so no need to keep it around